web-ui/src/app/components/latest-blocks/latest-blocks.component.ts

```diff
@@ -92,4 +115,8 @@ export class LatestBlocksComponent implements OnInit, OnDestroy {
   age(block: Block): string {
     return '';
   }
+
+  isBlockRecent(index: number): boolean {
```

Comment:
In my opinion, checking this by an index is error prone, angular allows you to pass the block object as an argument to the function, on the testability side, it would be simpler to receive both arguments instead of referencing a mutable variable.